telemetry/definitions/commonDefinitions.json

```
@@ -1287,6 +1292,11 @@
             "type": "int",
             "description": "Number of findings discovered after executing IAM Policy Checks"
         },
+        {
+            "name": "fixGenerated",
```

should this share a common prefix with the exsting `codeFix..` field? helps discoverability, assuming they are somewhat related.

https://github.com/aws/aws-toolkit-common/blob/ab466029f241193ab93a8abb3a5ca8dde5f56c94/telemetry/definitions/commonDefinitions.json#L286

```suggestion
            "name": "codeFixGenerated",
```